### PR TITLE
feat: use fonts

### PR DESCRIPTION
**Pull request by mkg20001**
Does this look good @Yutyo ?
Don't know if raleway bold for sansserif would look better

### CODE REVIEW

**Comment by mkg20001** [comment]
I mean if the design looks good with the new fonts.

Also trying to make lightdm use the cursor but doesn't work yet

File: config/base/branding.nix [hunk 1]
```diff
@@ -37,4 +38,12 @@ with lib;
   environment.pathsToLink = [
     "/share/wallpapers"
   ];
+
+  fonts.fonts = with pkgs; [
+    raleway
```
Comment (Yutyo) [inline config/base/branding.nix]: Raleway-Medium

File: config/base/branding.nix [hunk 2]
```diff
@@ -37,4 +38,12 @@ with lib;
...
+    jetbrains-mono
```
Comment (Yutyo) [inline config/base/branding.nix]: Jetbrains-Mono-Regular

Comment (mkg20001) [reply config/base/branding.nix]: These are the packages, the actual font names are further down below

File: config/base/branding.nix (hunk 1 above)
Comment (mkg20001) [reply config/base/branding.nix]: Doesn't seem to exist, the raleway pkg only ships regular
We'd need to package that

**Comment by mkg20001** [comment]
Applied requested changes

**Pull request merged**